Pusher.Connections.Net/WebSocketConnection.cs

```diff
@@ -6,132 +6,136 @@
 

```

Comment:
@vegardlarsen The changes in spacing should be resolved. Essentially just added a `try/catch` around the `Open` method to prevent a crash